airflow/ui/src/pages/Dag/Backfills/Backfills.tsx

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Heading, HStack, Text } from "@chakra-ui/react";
+import type { ColumnDef } from "@tanstack/react-table";
+import { MdOutlineContentCopy } from "react-icons/md";
+import { useParams } from "react-router-dom";
+
+import { useBackfillServiceListBackfills } from "openapi/queries";
+import type { BackfillResponse } from "openapi/requests/types.gen";
+import { DataTable } from "src/components/DataTable";
+import { useTableURLState } from "src/components/DataTable/useTableUrlState";
+import { ErrorAlert } from "src/components/ErrorAlert";
+import Time from "src/components/Time";
+import { reprocessBehaviors } from "src/constants/reprocessBehaviourParams";
+import { pluralize } from "src/utils";
+
+const columns: Array<ColumnDef<BackfillResponse>> = [
+  {
+    accessorKey: "id",
+    cell: ({ row }) => (
+      <HStack>
+        <Text>{row.original.id}</Text>
+        <MdOutlineContentCopy
+          onClick={() => {
+            void navigator.clipboard.writeText(row.original.id.toString());
+          }}
+        />
+      </HStack>
+    ),
+    enableSorting: true,
+    header: () => "ID",
+  },
+  {
+    accessorKey: "reprocess_behavior",
+    cell: ({ row }) => (
+      <Text>
+        {
+          reprocessBehaviors.find((rb: { value: string }) => rb.value === row.original.reprocess_behavior)
+            ?.label
+        }
+      </Text>
+    ),
+    enableSorting: false,
+    header: () => "Reprocess Behavior",
```

Comment:
```suggestion
    header: "Reprocess Behavior",
```

We haven't been enforcing this nit. But the function isn't necessary here.

Reply:
Ack